include/openssl/bio.h
=======================================================
lhs: 100644 | 03a6f04c754da2d152ea6c17f80eebefadef86d7
rhs: 100644 | f98f6ee11b8e67b8720f6354b25cdc0b5100cfe1
---@@ -809,12 +809,12 @@ int BIO_meth_set_callback_ctrl(BIO_METHOD *biom,
                                                       bio_info_cb *));
 
 /* BEGIN ERROR CODES */
+
 /*
- * The following lines are auto generated by the script mkerr.pl. Any changes
- * made after this point may be overwritten when the script is next run.
+ * Content after this point is generated by util/mkerr.pl
+ * DO NOT EDIT!
  */
 void ERR_load_BIO_strings(void);
-
 /* Error codes for the BIO functions. */
 
 /* Function codes. */
@@ -822,6 +822,7 @@ void ERR_load_BIO_strings(void);
 # define BIO_F_ADDR_STRINGS                               134
 # define BIO_F_BIO_ACCEPT                                 101
 # define BIO_F_BIO_ACCEPT_EX                              137
+# define BIO_F_BIO_ADDR_NEW                               144
 # define BIO_F_BIO_BER_GET_HEADER                         102
 # define BIO_F_BIO_CALLBACK_CTRL                          131
 # define BIO_F_BIO_CONNECT                                138

---